### PR TITLE
Avoid printing cert message for already loaded route

### PR DESCRIPTION
**Pull request by imcsk8**
Theres a certificate message that is being written to the log every
time the router reloads it's configuration, this message is only
useful when the route is being created or modified.

Fixes BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1401503

### CODE REVIEW

File: pkg/router/template/router.go [hunk 1]
```diff
@@ -785,6 +785,12 @@ func cmpStrSlices(first []string, second []string) bool {
 // it will log a warning.  The route will still be written but users may receive browser errors
 // for a host/cert mismatch
 func (r *templateRouter) shouldWriteCerts(cfg *ServiceAliasConfig) bool {
+
+	// The cert is already written
+	if cfg.Status == ServiceAliasConfigStatusSaved {
```
Comment (knobunc) [inline pkg/router/template/router.go]: Do we ever set the status to this when there are no certs?  We return false if they don't have certs, and that means we never call WriteCertificatesForConfig.

So, do we need to set cfg.Status in this function?  If so, should we re-use the same state or make a new one "Skipped" or something?

Comment (imcsk8) [inline pkg/router/template/router.go]: The status is not set unless there's an error writing the certificate (https://github.com/openshift/origin/blob/master/pkg/router/template/router.go#L427).
If we really need to be explicit, a new state can be created here (https://github.com/openshift/origin/blob/master/pkg/router/template/types.go#L62)

**Comment by knobunc** [comment]
[test]

**Comment by imcsk8** [comment]
[test]

**Comment by knobunc** [comment]
[test] Last flaked with https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin/480/consoleFull#-96647656556cbb9a5e4b02b88ae8c2f77 which is #13536

**Comment by knobunc** [comment]
[test] (the old failure had timed out, so I can't diagnose)

**Comment by knobunc** [comment]
[merge]

**Comment by knobunc** [comment]
[merge] -- The last flaked because we couldn't provision cloud resources (https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin/882/)

**Pull request merged**